src/Libplanet.Types/Assets/Currency.cs

```diff
@@ -742,40 +728,56 @@ private static SHA1 GetSHA1()
 #endif
         }
 
-        [Pure]
-        private IValue SerializeForHash()
+        private static IValue SerializeForHash(
+            IImmutableSet<Address>? minters,
+            string ticker,
+            byte decimalPlaces,
+            (BigInteger Major, BigInteger Minor)? maximumSupply,
+            bool totalSupplyTrackable)
```

Comment:
Is it necessary?

Reply:
I think it's not, That code has been moved into `GetHash`.
amended https://github.com/planetarium/libplanet/pull/3880/commits/997922be4554673eba597395fe03f1efee5f4d6c

Reply:
Wasn't there a reason why `GetHash()` and `SerializeForHash()` were separate in the first place? 🤔

Reply:
Ah, nevermind, it was about `SerializeForHash()` and `Serialize()`. 🙄

Reply:
It might be good to have a simple comment on whether we have a separate serialization scheme. 😐

Reply:
> It might be good to have a simple comment on whether we have a separate serialization scheme. 😐

https://github.com/planetarium/libplanet/pull/3880/commits/3808b22190d2bf242c44416b47315184856ba235#diff-7975b57b58acd250c0c5fb4f0001c98e298746481661b38ccb8e64c429dda7b5R731
How about this?

Reply:
The comment doesn't clarify as to **why** the `Dictionary` construction is different from `Serialize()`, which I think would be the main point of confusion for those seeing the code without context. I'd say the history regarding `GetHash()` is barely relevant and doesn't need mentioning.

Reply:
> The comment doesn't clarify as to **why** the `Dictionary` construction is different from `Serialize()`, which I think would be the main point of confusion for those seeing the code without context. I'd say the history regarding `GetHash()` is barely relevant and doesn't need mentioning.

https://github.com/planetarium/libplanet/pull/3880/commits/9a8fd893bb89c127168ba77cfbb223123d0122c9#diff-7975b57b58acd250c0c5fb4f0001c98e298746481661b38ccb8e64c429dda7b5R731
amended